packages/legend-query/src/QueryBuilder_Const.ts

```diff
@@ -70,6 +84,7 @@ export enum SUPPORTED_FUNCTIONS {
   STD_DEV_POPULATION = 'meta::pure::functions::math::stdDevPopulation',
   STD_DEV_SAMPLE = 'meta::pure::functions::math::stdDevSample',
   SUM = 'meta::pure::functions::math::sum',
+  MINUS = 'meta::pure::functions::math::minus',
```

Comment:
`MINUS` is not part of `aggregation` so let's move `MINUS` and `ADJUST` to their separate group for now,

```ts
...

// adjust time
ADJUST = 'meta::pure::functions::date::adjust',
MINUS = 'meta::pure::functions::math::minus',

...
```